src/main/java/at/ac/tuwien/damap/rest/persons/MockPersonRestService.java

```diff
@@ -1,25 +1,44 @@
 package at.ac.tuwien.damap.rest.persons;
 
-import at.ac.tuwien.damap.rest.dmp.domain.ContributorDO;
-import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+import java.net.URL;
+import java.util.List;
 
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
-import java.util.List;
+
+import org.eclipse.microprofile.rest.client.RestClientBuilder;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import at.ac.tuwien.damap.rest.dmp.domain.ContributorDO;
+import io.quarkus.restclient.config.RestClientsConfig;
 
 @Path("")
-@RegisterRestClient(configKey = "rest.persons")
+// needed for inject annotations
+@RegisterRestClient(configKey = MockPersonRestService.configKey)
```

Comment:
blocking this construction should not be necessary, since this external rest service is only used by mock implementation.